app/assets/javascripts/app/controllers/recovery_page_controller.js

```diff
@@ -0,0 +1,93 @@
+import Modal from '../components/modal';
```

Comment:
the naming of this file confuses me -- are we starting a new convention of naming .js files for the controllers/views where they are used?